js/main.js

```diff
@@ -35,6 +42,18 @@ Site = {
   onResize: function() {
     var _this = this;
 
+    _this.resizeDelay();
+  },
+
+  resizeDelay: function() { 
```

Comment:
he didnt write this func. John Hann did. read the post. and this func does essentially what im already doing but also expands jquery by adding a `$(elem).smartresize()` and an option for `execAsap` which we dont need.